modules/microshift-creating-backups.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * microshift/microshift_backup_and_restore/microshift-auto-recover-manual-backup.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-creating-backups_{context}"]
+= Creating backups using the auto-recovery feature
+
+You can use the following procedure to create backups.
+
+[NOTE]
+====
+Creating backups require stopping {microshift-short}, you must determine the best time to stop {microshift-short}.
+====
+
+.Prerequisites
+
+* You have stopped {microshift-short}.
+
+.Procedure
+
+. To store backups, create a backup in the directory you choose by running the following command:
++
+[source,terminal]
+----
+$ sudo microshift backup --auto-recovery _<path_of_directory>_ <1>
+----
+<1> Replace `_<path_of_directory>_` with the path of the directory that stores backups. For example, `/var/lib/microshift-auto-recovery`.
++
+[NOTE]
+====
+The `--auto-recovery` option modifies the interpretation of the `PATH` argument from the final backup path to a directory that holds all the backups for auto-recovery.
+====
++
+.Example output
++
+[source,terminal]
+----
+$ sudo microshift backup --auto-recovery /var/lib/microshift-auto-recovery
```

Comment:
If this is example output then it should only contain the output, and the example command should be removed:
```suggestion
```

Reply:
done